package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "amcharts3-react",
-  "version": "1.1.8",
+  "version": "1.1.9",
   "license": "SEE LICENSE IN LICENSE",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/amcharts/amcharts3-react.git"
+  },
+  "dependencies": {
+    "lodash": "^4.17.4"
```

Comment:
I'd rather not have a dependency on `lodash`, especially because it's not needed (see comments above).